src/network/udp.py

```diff
@@ -21,7 +21,7 @@ class UDPSocket(BMProto):  # pylint: disable=too-many-instance-attributes
     port = 8444
 
     def __init__(self, host=None, sock=None, announcing=False):
-        # pylint: disable=bad-super-call
+        # pylint: disable=bad-super-call, access-member-before-definition
```

Comment:
This is a bug: https://github.com/PyCQA/pylint/issues/2315

Reply:
The linked bug has been fixed. The ignore shouldn't be needed anymore.